packages/core/src/dialects/cockroachdb/query-interface.js

```diff
@@ -0,0 +1,30 @@
+import { PostgresQueryInterface } from '../postgres/query-interface';
+
+export class CockroahcDbQueryInterface extends PostgresQueryInterface {
```

Comment:
note that all new files should be written in TypeScript instead of JS

We have a bunch of .js + .d.ts files in the codebase because it's a very old codebase that we're slowly migrating to TypeScript

Reply:
Thanks will do that!